com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -40,7 +40,8 @@ VertexNormalInputs GetVertexNormalInputs(float3 normalOS, float4 tangentOS)
 
 float4 GetScaledScreenParams()
 {
-    return _ScaledScreenParams;
+    // NOTE: Builtin does not use _ScaledScreenParams, so we simply return _ScreenParams instead
+    return _ScreenParams;
```

Comment:
Thanks for the comment here.  This kind of info gets lost easily (or is never found).